RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -238,7 +265,9 @@ - (NSDictionary *)constantsToExport
              @"isEmulator": @(self.isEmulator),
              @"isTablet": @(self.isTablet),
              @"is24Hour": @(self.is24Hour),
-             @"totalMemory": @(self.totalMemory)
+             @"totalMemory": @(self.totalMemory),
+			 @"totalDiskCapacity": @(self.totalDiskCapacity),
+			 @"freeDiskStorage": @(self.freeDiskStorage),
```

Comment:
whitespaces

Reply:
Sure, will fix them and update README.md